apps/mull-ui/src/app/pages/messages/announcements/announcements.tsx

```diff
@@ -1,71 +1,45 @@
-        or otherwise generated. It may be used to display a sample of fonts, generate text for
-        testing, or to spoof an e-mail spam filter.
-      </ChatBubble>
-
-      <ChatBubble
-        isCurrentUser={true}
-        chatDate="13:00 21/10/20"
-        userPicture="https://blog.photofeeler.com/wp-content/uploads/2017/04/are-bumble-profiles-fake-how-many.jpeg"
-      >
-        Lorem ipsum is placeholder text commonly used in the graphic, print, and publishing
-        industries for previewing layouts and visual mockups.
-      </ChatBubble>
-      <ChatBubble
-        isCurrentUser={true}
-        chatDate="13:00 21/10/20"
-        userPicture="https://blog.photofeeler.com/wp-content/uploads/2017/04/are-bumble-profiles-fake-how-many.jpeg"
-      >
-        Lorem ipsum is placeholder text commonly used in the graphic, print, and publishing
-        industries for previewing layouts and visual mockups.
-      </ChatBubble>
-      <ChatBubble
-        isCurrentUser={true}
-        chatDate="13:00 21/10/20"
-        userPicture="https://blog.photofeeler.com/wp-content/uploads/2017/04/are-bumble-profiles-fake-how-many.jpeg"
-      >
-        Lorem ipsum is placeholder text commonly used in the graphic, print, and publishing
-        industries for previewing layouts and visual mockups.
-      </ChatBubble>
-      <ChatBubble
-        isCurrentUser={true}
-        chatDate="13:00 21/10/20"
-        userPicture="https://blog.photofeeler.com/wp-content/uploads/2017/04/are-bumble-profiles-fake-how-many.jpeg"
-      >
-        Lorem ipsum is placeholder text commonly used in the graphic, print, and publishing
-        industries for previewing layouts and visual mockups.
-      </ChatBubble>
-      <ChatBubble
-        isCurrentUser={true}
-        chatDate="13:00 21/10/20"
-        userPicture="https://blog.photofeeler.com/wp-content/uploads/2017/04/are-bumble-profiles-fake-how-many.jpeg"
-      >
-        Lorem ipsum is placeholder text commonly used in the graphic, print, and publishing
-        industries for previewing layouts and visual mockups.
-      </ChatBubble>
-      <ChatInput />
-    </div>
+    <>
+      <ChatBubbleList
+        data={{
+          posts: [
+            {
+              user: {
+                id: 1,
+                avatar: null,
+              },
+              message: 'message 1 from user 1, current user id is 1',
+              createdTime: new Date(),
+            },
+            {
+              user: {
+                id: 2,
+                avatar: null,
+              },
+              message: 'message 2 from user 2',
+              createdTime: new Date(),
+            },
+            {
+              user: {
+                id: 3,
+                avatar: { id: 1 },
+              },
+              message: 'message 3 from user 3',
+              createdTime: new Date(),
+            },
+          ],
+          currentUser: {
+            id: 1,
+          },
```

Comment:
This shouldn't be passed, the ChatBubbleList can get the currentUser from context. See `private-route.tsx` for an example
```suggestion
```